From 40394cb1c963c35d2daa9570ba968afa981cf1fc Mon Sep 17 00:00:00 2001
From: Euan Harris <euan.harris@docker.com>
Date: Thu, 3 May 2018 13:34:46 +0100
Subject: [PATCH 1/2] nsenter: Rename --network option to --net

In nsenter from util-linux, the long version of the -n option is
--net=<path>.   BusyBox's version expects --network=<path>, so scripts
and examples written for util-linux's version cause BusyBox's version
to exit with the usage message.

Confusingly, until commit 036585a911a5f, the usage message erroneously
claimed that the long option was indeed called --net; after that commit
long options are not listed at all.

Signed-off-by: Euan Harris <euan.harris@docker.com>
Signed-off-by: Denys Vlasenko <vda.linux@googlemail.com>
---
 util-linux/nsenter.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util-linux/nsenter.c b/util-linux/nsenter.c
index c6933c8..35439a2 100644
--- a/util-linux/nsenter.c
+++ b/util-linux/nsenter.c
@@ -112,7 +112,7 @@ static const char nsenter_longopts[] ALIGN1 =
 	"user\0"			Optional_argument	"U"
 	"ipc\0"				Optional_argument	"i"
 	"uts\0"				Optional_argument	"u"
-	"network\0"			Optional_argument	"n"
+	"net\0"				Optional_argument	"n"
 	"pid\0"				Optional_argument	"p"
 	"mount\0"			Optional_argument	"m"
 	"target\0"			Required_argument	"t"
-- 
2.7.4

